.sass-lint.yml

```diff
@@ -0,0 +1,220 @@
+linters:

```

Comment:
I have issue with linter plugin on my Atom, can not figure out what's causing it, but creation of this file fixes it.